src/deployconfig.jl

```diff
@@ -104,6 +104,18 @@ marker(x) = x ? "✔" : "✘"
 
 env_nonempty(key) = !isempty(get(ENV, key, ""))
 
+
+"""
+    CommonDevBranches
+
+A stand-in for the name of the devbranch. Rather than hardcoding to `master` or `main` this
+accepts any of `master`, `main`.
+"""
+struct CommonDevBranches end
+
+matches_devbranch(branchname, devbranch) = branchname == devbranch
+matches_devbranch(branchname, ::CommonDevBranches) = branchname ∈ ("main", "master", "dev")
```

Comment:
```suggestion
matches_devbranch(branchname, ::CommonDevBranches) = branchname ∈ ("main", "master")
```

Reply:
Before I was considering also allowing `dev`.
But the only repo i know that uses `dev` is MLJ, and they do not see the `devbranch=dev` argument.
And so accepting that would potentially break their workflow.
https://github.com/alan-turing-institute/MLJ.jl/blob/96562a60a70aebc3f0d768147ff9dab4549ddfa5/docs/make.jl#L98-L101